### PR TITLE
HARP-12995 Fix flickering of new markers

### PR DESCRIPTION
**Pull request by nzjony**

### CODE REVIEW

**Comment by nzjony** [comment]
retest this please

**Comment by nzjony** [comment]
retest this please

File: @here/harp-mapview/lib/VisibleTileSet.ts [hunk 1]
```diff
@@ -1313,9 +1313,6 @@ export class VisibleTileSet {
             const tileKey = DataSourceCache.getKeyForTile(tile);
             if (!retainedTiles.has(tileKey)) {
                 retainedTiles.add(tileKey);
-                // Prevent label rendering issues when the style set is changing. Prevent Text
-                // element rendering that depends on cleaned font catalog data.
-                tile.clearTextElements();
```
Comment (FraukeF) [inline @here/harp-mapview/lib/VisibleTileSet.ts:1318]: Would it make sense to actually clear the whole tile at this point, tile.clear() as its going to be reloaded/created

Comment (nzjony) [reply @here/harp-mapview/lib/VisibleTileSet.ts:1318]: We want to avoid flickering, hence marking the tile as dirty and reloading it.

Comment (FraukeF) [reply @here/harp-mapview/lib/VisibleTileSet.ts:1318]: ah right ,sorry, I read this the wrong way

**Review by atomicsulfate — APPROVED** [review]

**Pull request merged**